packages/jest-cli/src/TestRunner.js

```diff
@@ -289,9 +323,40 @@ const addResult = (
   } else {
     aggregatedResults.numPassedTestSuites++;
   }
+
+  // Snapshot data
+
+
+  if (testResult.snapshot.added) {
+    aggregatedResults.snapshot.filesAdded++;
```

Comment:
No, I think this is correct. If a snapshot was added in this test, the number of filesAdded increases by one, regardless of how many snapshots there are.